website/docs/d/cloud_provider_shared_tier_restore_job.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_shared_tier_restore_job"
+sidebar_current: "docs-mongodbatlas-datasource-mongodbatlas-shared-tier-restore-jobs"
+description: |-
+    Provides a Cloud Backup Shared Tier Snapshot Restore Job Datasource.
+---
+
+
+# Data Source: mongodbatlas_shared_tier_restore_job
+
+`mongodbatlas_shared_tier_restore_job` provides a Cloud Backup Snapshot Restore Jobs datasource for Shared Tier Clusters. Gets the cloud backup snapshot restore jobs for the specified shared tier cluster.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+-> **NOTE:** This datasource is only for Shared Tier Clusters (M0, M2, M5).
```

Comment:
```suggestion
-> **NOTE:** This data source is only for Shared Tier Clusters (M2 and M5). See [here](https://www.mongodb.com/docs/atlas/reference/free-shared-limitations/) more details on Shared Tier Cluster Limitations. 
```

Reply:
Note M0 is not supported by this API endpoint - https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Shared-Tier-Snapshots/operation/getSharedClusterBackup